core/test_network/src/lib.rs

```diff
@@ -355,7 +344,7 @@ where
 /// When unsuccessful after `MAX_RETRIES`.
 pub fn wait_for_genesis_committed(clients: &[Client], offline_peers: u32) {
     const POLL_PERIOD: Duration = Duration::from_millis(1000);
-    const MAX_RETRIES: u32 = 60 * 3; // 3 minutes
+    const MAX_RETRIES: u32 = 10; // 3 minutes
```

Comment:
And the comment is still valid, because we're retrying every 18 seconds.

Reply:
Comment was not still valid, so I removed the comment.